pkgs/development/python-modules/plugincode/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, setuptools-scm
+, click
+, commoncode
+, pluggy
+, pytest
+, pytest-xdist
+}:
+buildPythonPackage rec {
+  pname = "plugincode";
+  version = "21.1.21";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "97b5a2c96f0365c80240be103ecd86411c68b11a16f137913cbea9129c54907a";
+  };
+
+  dontConfigure = true;
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
+  propagatedBuildInputs = [
+    click
+    commoncode
+    pluggy
+  ];
+
+  checkInputs = [
+    pytest
```

Comment:
```suggestion
    pytestCheckHook
```

Reply:
Missed this one, thanks.

Also added `dontStrip`, because you're right. Closure size didn't change:

```shell
$ nix path-info -Sh ./result # dontStrip = false;
/nix/store/pdw522f5ali0dkn57c423sljc3gxv7ha-python3.8-scancode-toolkit-21.3.31   486.3M
$ nix path-info -Sh ./result # dontStrip = true;
/nix/store/abc2kslllbpl69kqgs6rby6qx96zma41-python3.8-scancode-toolkit-21.3.31   486.3M
```